### PR TITLE
Pre release tweeks

### LINKED ISSUE #104: Blog posts do not load on first visit from blog home page

### PR DESCRIPTION
**Pull request by KaneFreeman**
## Description

- Footer / Sidemenu CSS tweek
- Decrease font-size to 14px
- Add table css styling for desktop and mobile setups
- Fixed issue with no-JS ordering of blog posts by changing the order of BTR routes.
- Fixed #104 by making `blog-post` a child route of `blog`.

### Code

This PR touches:

- [ ] Content
- [ ] Content Pipeline
- [x] Frontend
- [ ] Infrastructure

### Tests

- [x] Tests are included?

### Screenshots

image

image

### CODE REVIEW

File: src/App.spec.tsx [hunk 1]
```diff
@@ -48,7 +56,8 @@ describe('App', () => {
 
 	const pages: Page[] = [
 		{ outlet: 'home', content: <Home /> },
-		{ outlet: 'blog', content: <Blog /> },
+		{ outlet: 'blog', content: <Blog />, args: [{ isExact: () => true }] },
+		{ outlet: 'blog', content: undefined, args: [{ isExact: () => false }] },
```
Comment (bitpshr) [inline src/App.spec.tsx]: Why do we need this inexact outlet for `blog` with no `content`? What does this do?

File: src/main.css [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 html {
 	font-family: 'Montserrat', sans-serif;
-	font-size: 15px;
+	font-size: 14px;
```
Comment (bitpshr) [inline src/main.css]: Are there any plans to pull out all these numerical values for things like spacing and font sizes into variables?

File: src/pages/reference-guides/ReferenceGuideMenu.tsx [hunk 3]
```diff
@@ -71,7 +71,7 @@ export default class ReferenceGuideMenu extends I18nMixin(WidgetBase)<ReferenceG
 
 		return (
 			<SideMenu>
-				<SideMenuSection>{list}</SideMenuSection>
+				<SideMenuSection fixed={true}>{list}</SideMenuSection>
```
Comment (bitpshr) [inline src/pages/reference-guides/ReferenceGuideMenu.tsx]: Nit: This should be changed to `fixed` or other places should be changed to `fixed={true}` for consistency (I prefer the former.)

File: src/widgets/menu/SideMenuSection.m.css [hunk 4]
```diff
@@ -4,6 +4,10 @@
 	margin-bottom: calc(var(--grid-size) * 3);
 }
 
+.root.fixed {
```
Comment (bitpshr) [inline src/widgets/menu/SideMenuSection.m.css]: Do you need this adjoining class? I think `fixed` alone would suffice.

Comment (bitpshr) [inline src/widgets/menu/SideMenuSection.m.css]: I don't think the adjoining `.root` class is necessary here.

File: src/widgets/page/LocalPage.tsx [hunk 5]
```diff
@@ -8,13 +8,12 @@ import Page from './Page';
...
 export interface LocalPageProperties {
 	path: string;
-	hasLeftSideMenu?: boolean;
 	warpInPage?: boolean;
 }
 
 export default class LocalPage extends WidgetBase<LocalPageProperties> {
 	protected render() {
-		const { path, hasLeftSideMenu = false, warpInPage = true } = this.properties;
+		const { path, warpInPage = true } = this.properties;
```
Comment (bitpshr) [inline src/widgets/page/LocalPage.tsx]: I think this should be `wrapInPage` (although `warpInPage` sounds pretty cool, so carry on if it's intentional ✊)

File: src/App.spec.tsx (hunk 1 above)
Comment (KaneFreeman) [reply src/App.spec.tsx]: By moving the individual blog page's route under `blog`, the parent gets triggered. So we need to check for the route to be exact.

File: src/main.css (hunk 2 above)
Comment (KaneFreeman) [reply src/main.css]: Good point.

File: script.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
+'use strict';
+/**
+ * Load the TypeScript compiler, then load the TypeScript gulpfile which simply loads all
```
Comment (matt-gadd) [inline script.js]: not sure if this file is meant to be in this PR, but either way the comment is copy/pasta'd from somewhere else 😄.

Comment (KaneFreeman) [reply script.js]: haha, yeah. will remove :P

**Pull request merged**